style.scss

```diff
@@ -1,19 +1,20 @@
+
+@import "colors";
+
 * {
     box-sizing: border-box;
 }
-@import "colors";
 
 html,
 body {
     height: 100%;
-    margin: 0;
+    margin: 0px;
```

Comment:
Please don't use units if the value is 0. Zero is always zero, no matter `px`, `%` or other units.